config/default.go

```diff
@@ -12,15 +12,15 @@ const DefaultValues = `
 PrivateKey = {Path = "/pk/test-member.keystore", Password = "testonly"}
 
 [L1]
-WsURL = "ws://127.0.0.1:8546"
-RpcURL = "http://127.0.0.1:8545"
+RpcURL = "ws://127.0.0.1:8546"
```

Comment:
It could be http(s) as well.

Reply:
EVM API uses the same handlers for ws and http protocols. You can dial ethclient with http or ws and it is going to have exactly the same behaviour. The difference is that http does not support subscriptions such as listening for new heads/blocks/events. 

Currently, only sequencer tracker uses ws, other places need http. I've updated the sequencer tracker to support http as well.